pkg/ddevapp/addons.go

```diff
@@ -131,17 +133,27 @@ func ProcessAddonAction(action string, dict map[string]interface{}, bashPath str
 		action = "set -x; " + action
 	}
 	out, err := exec.RunHostCommand(bashPath, "-c", action)
-	if len(out) > 0 {
-		util.Warning(out)
-	}
+
 	if err != nil {
-		util.Warning("%c %s", '\U0001F44E', desc)
-		return fmt.Errorf("Unable to run action %v: %v, output=%s", action, err, out)
-	}
-	if desc != "" {
+		var exitErr *goexec.ExitError
+		if errors.As(err, &exitErr) {
+			// Get the exit code
+			exitCode := exitErr.ExitCode()
+			if exitCode == 63 {
```

Comment:
Is there a reason for `63` specifically?
Do we have somewhere sensible we can set this as a constant so it's not just a magic number floating around?

Reply:
This is what anthropic had to say about this:

I went for 63 as in the higher application-specific error code based on that but
1. it could be anything
2. I can't say if that is right

> ## Warning Handling Approaches
> 
> While there's no standard "warning" exit code, some applications follow these patterns:
> 
> 1. **Non-Zero but Documented**: Some tools use specific exit codes like **1** or **2** for warnings and document this behavior
>    
> 2. **LSB (Linux Standard Base)** suggests ranges:
>    - 1-63: Application-specific error codes
>    - 64-113: More specific errors (sometimes following `sysexits.h`)
> 
> 3. **sysexits.h** defines error codes (64-78) for specific situations but doesn't include warnings specifically